File: ProductsAPI/ProductsAPI/Models/Product.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProductsAPI.Models
{
    public partial class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public int CategoryId { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedAt { get; set; }
        public bool IsArchived { get; set; }

        public virtual Category Category { get; set; }
        public virtual Person CreatedByNavigation { get; set; }
    }
}

File: front/src/app/product-table/product.ts
export interface Product{
  id: number;
  name: string;
  description: string;
  price: number;
  category_id: number;
  created_by_id: number;
  category_name: string;
  created_by_name: string;
  created_at: string;
  is_archived: boolean;
  chosen_to_be_archived: boolean;
  being_edited: boolean;
  shown: boolean;
}